internal/generator/vector/output/elasticsearch/passphrase.go

```diff
@@ -0,0 +1,18 @@
+package elasticsearch
+
+import (
+	"github.com/openshift/cluster-logging-operator/internal/generator/vector/output/security"
+)
+
+type Passphrase security.Passphrase
+
+func (p Passphrase) Name() string {
+	return "passphraseTemplate"
+}
+
+func (p Passphrase) Template() string {
+	return `{{define "` + p.Name() + `" -}}
+key_pass = "default"
```

Comment:
`key_pass` is Pass phrase used to unlock the encrypted key file. 

perhaps we can delete the PassPhrase from ES vector config

Reply:
Ok make sense.